### PR TITLE
Clean up after 20.08.3 release

### PR DESCRIPTION
**Pull request by swaterkamp**
**What**:
Set versions accordingly and add empty unreleased section to changelog

**Why**:

**How**:

**Checklist**:

- [ ] Tests
- [ ] [CHANGELOG](https://github.com/greenbone/gsa/blob/master/CHANGELOG.md) Entry
- [ ] Labels for ports to other branches

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
...
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+[Unreleased]: https://github.com/greenbone/gsa/compare/v20.8.3...v20.8.2
```
Comment (y0urself) [inline CHANGELOG.md:18]:
```suggestion
[Unreleased]: https://github.com/greenbone/gsa/compare/v20.8.4...v21.4.0
```

**Review by y0urself — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (swaterkamp) [reply CHANGELOG.md:18]: This produces a broken link, doesn't it? In the past we had at least a working link here and fixed it when releasing

Comment (swaterkamp) [reply CHANGELOG.md:18]: And why should be compare it to 21.4.0?

Comment (y0urself) [reply CHANGELOG.md:18]: ahh I did it wrong ... wait ...

Comment (y0urself) [reply CHANGELOG.md:18]: nvm ... now it should be correct ... I guess?

**Review by y0urself — APPROVED** [review]

**Pull request merged**